extras/anthos-vmruntime/README.md

```diff
@@ -54,11 +54,13 @@ Environment=SPRING_PROFILES_ACTIVE=inmemory
     ```
 
 - Make the bucket **publicly readable**
+    > NOTE: For typical use the bucket should be **private** and [credentials](https://cloud.google.com/anthos/clusters/docs/bare-metal/latest/vm-runtime/create-storage-credentials) used to import the image into VM Runtime.
```

Comment:
I am not sure I understand this sentence 🤔 

![image](https://user-images.githubusercontent.com/7249208/218184724-e788ba78-29df-4046-abcb-0aee26f75068.png)

Is it trying to say:

`.... the bucket and the credentials should be provate?`

Reply:
Can we format the `NOTE` as follows:

`> **Note**:`

> **Note:**

Reply:
I rewrote to hopefully be clearer.  The GCS bucket should be private.  Then credentials can be configured in VM Runtime to access the image from the private bucket.